### ISSUE TITLE
About LOLv2_real configuration

### ISSUE THREAD

**Issue by Sttrr** [wangchx67/FourLLIE#16]
Hello, we use the config file in the folder 'options/train/LOLv2_real.yml' to retrain the model, but the results obtained are a little worse than the ones presented in the paper. I dont know what i did wrong, Can you give me some help? my dataset and config with the results are presented below:
![捕获](https://github.com/wangchx67/FourLLIE/assets/38433751/433aefa0-8eb6-496b-989b-10f29f6be712)
![捕获2](https://github.com/wangchx67/FourLLIE/assets/38433751/bf88f2ba-d370-4a52-ab9d-4dfac7bd2984)
![捕获3](https://github.com/wangchx67/FourLLIE/assets/38433751/075f22c2-15fb-45cc-9a81-c3d4cb358d5e)

**Comment by wangchx67** [comment]
Hello,

Apologies for the delayed response. I've cloned the repository and retrained the model locally, and it successfully achieves the similar results outlined in the paper. You may wish to experiment with setting the GT_size and LQ_size to 384. Additionally, the seed used for my reproduction is 7335, so you can try to set your manual_seed to 7335 as well.

Besides, you also can directly use our pre-trained model in `./pre-trained/lol2-real.pth`.

Best regards,

**Comment by Sttrr** [comment]
Thans for your response,

I modified the GT_size and random seed as you suggested, and it did improve. However, it is still a little worse than the results outlined in the paper. My best results were around 20000 iterations and slightly lower thereafter. Is that normal? I want to use your network in my work so i hope to retrain it. Can you send me your training log? It will be useful for me. My Gmail address is liubo3949@gmail.com
![捕获](https://github.com/wangchx67/FourLLIE/assets/38433751/c6295d95-b979-461d-b351-48c7099d215c)

**Comment by wangchx67** [comment]
please check your email

**Comment by Newbie-v** [comment]
Could you please send me a copy of the training logs? Thank you.My Gmail address is niubin1216@gmail.com

> 请检查您的电子邮件

**Comment by wangchx67** [comment]
check the email

**Comment by bin-boss** [comment]
Hello, I'm experiencing the same issue. The best metrics for model retrained on the LOLv2_real dataset are shown below. They differ slightly from the metrics reported in your paper. I'd like to know the cause of this discrepancy and how to adjust it to perfectly replicate the results in your paper. Also, could you provide the relevant training logs?My Gmail address is bin41470@gmail.com
Thank you.

<img width="1192" height="101" alt="Image" src="https://github.com/user-attachments/assets/2a37fc83-c28d-4fa0-aab6-76b7e7a93f00" />
